Curriculum updates for HPC Carpentry adoption (#2020)
carpentries/instructor-training: episodes/15-carpentries.md

- [Software Carpentry: Lessons Learned][F1000]
- [Data Carpentry: Workshops to Increase Data Literacy for Researchers][IJDC]
- [Expanding user communities with HPC Carpentry][JOSCE]
- [Library Carpentry: software skills training for library professionals][LIBERQ]

## Similarities and Differences between The Carpentries Lesson Programs

All lesson programs under The Carpentries share the same [core values][values-page].
Their aligned missions are accomplished by running accessible, inclusive training workshops; teaching openly available, high-quality, community-developed lessons; and fostering an active, inclusive, diverse instructor community that promotes and models reproducible research as a community norm.

Similarities between Data Carpentry, HPC Carpentry, Library Carpentry, and Software Carpentry workshops include:

- a focus on technical skills,
- a two full day or four half day format taught by volunteer instructors, and
- a focus on filling gaps in current training for learners.

The major differences between Data Carpentry, HPC Carpentry, Library Carpentry, and Software Carpentry workshops are their content and intended audience. 

> Their similarities and differences are depicted in this diagram:

Reviewer: ```suggestion
Their similarities and differences are depicted in the following table:
```

or should we not call it a table since it's technically an image?
